Topic1_Arrays/Day2207/3.txt

```diff
@@ -0,0 +1,15 @@
+var merge = function (nums1, m, nums2, n) {
```

Comment:
Em thêm phần đánh giá độ phức tạp thuật toán vào trong các bài sau cho quen nhé. 
Sau này đi phỏng vấn coding interview sẽ hay bị hỏi.